src/Project.ts

```diff
@@ -59,8 +61,11 @@ export class Project {
         const tsConfigResolver = options.tsConfigFilePath == null ? undefined : new TsConfigResolver(fileSystemWrapper, options.tsConfigFilePath, getEncoding());
         const compilerOptions = getCompilerOptions();
 
+        // init compiler host overrides
+        const compilerHost = options.compilerHostFactoty ? options.compilerHostFactoty(fileSystem) : {};
```

Comment:
The best part about TypeScript is how it helps developers consistently misspell a word ;) (Re: `Factoty`)